packages/client-config/API.md

```diff
@@ -627,7 +627,7 @@ export type CustomClientConfig = {
 export const DEFAULT_CLIENT_CONFIG_VERSION: ClientConfigVersion;
 
 // @public
-export const generateClientConfig: <T extends "1" | "1.1" | "1.2" | "1.3" | "0">(backendIdentifier: DeployedBackendIdentifier, version: T, awsClientProvider?: AWSClientProvider<{
+export const generateClientConfig: <T extends "1.2" | "1.3" | "1.1" | "1" | "0">(backendIdentifier: DeployedBackendIdentifier, version: T, awsClientProvider?: AWSClientProvider<{

```

Comment:
could you please run `npm run clean && npm install && npm run build && npm run update:api` ?
or simply revert this change ?
this is an unwanted diff which sometimes happens in incrementally built local workspace.

Reply:
Updated in [091f133](https://github.com/aws-amplify/amplify-backend/pull/2269/commits/091f133b18a0bc04843ef8ed6b39d46efe550ce1)